### PR TITLE
Cart Page Added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Below elements added :

1. Quantity of a particular product.
2. Remove  and Add button 
3. Price details

### Link For Review
https://e-com-cart.netlify.app/pages/cart

If Home Page is popping up then click on cart icon on navbar

### CODE REVIEW

**Review by shaikhhamza11 — APPROVED** [review]
Lgtm

File: Pages/cart.html [hunk 1]
```diff
@@ -17,7 +17,125 @@
...
+                <span class="md-text">$220</span>
+            </div>
+            <hr class="line">
+            <h3 class="sm-text">You will save $199 on this Order</h3>
+            <button class="btn-primary btn">Place Order</button>
+       </div>
+   </div>
+
+    <!-- Footer  -->
+    <div id="footer">
```
Comment (adith-a-danthi) [inline Pages/cart.html:127]: You can use `footer` tag here

File: Pages/product.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h1 class="brand-name lg-h">Apani Dukan</h1>
                 <div class="number sm-text">4</div>
             </div>
             <div class="badge nav-right-ele">
-                <i class="fas fa-shopping-cart lg-text"></i>
+                <i class="fas fa-shopping-cart lg-text" onclick="location.href='/Pages/cart.html'"></i>
```
Comment (adith-a-danthi) [inline Pages/product.html:37]: This can probably be done using an anchor tag instead.

File: CSS/cart.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
+#cart-container{
```
Comment (adith-a-danthi) [inline CSS/cart.css:1]: This can just be a class selector I think

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good. Clean code and good use of comments!

**Review by sakleprajwal — APPROVED** [review]
Everything looks good. Please go ahead.

**Pull request merged**